cmake/TestImpactFramework/TestImpactTestTargetConfig.cmake

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+# Path to test instrumentation binary
+set(O3DE_TEST_IMPACT_INSTRUMENTATION_BIN "" CACHE PATH "Path to test impact framework instrumentation binary")
+
+# Test impact analysis opt-in for native test targets
+set(O3DE_TEST_IMPACT_NATIVE_TEST_TARGETS_ENABLED FALSE CACHE BOOL "Whether to enable native C++ test targets for test impact analysis (otherwise, CTest will be used to run these targets).")
+
+# Test impact analysis opt-in for Python test targets
+set(O3DE_TEST_IMPACT_PYTHON_TEST_TARGETS_ENABLED FALSE CACHE BOOL "Whether to enable Python test targets for test impact analysis (otherwise, CTest will be used to run these targets).")
```

Comment:
Yes, but as 

> LGTM! Just to make sure I understand correctly this makes it super simple/easy to enable/disable TIAF and switch back to CTest (if there's any issue that pops up for example). Otherwise looks solid!

Exactly that 👍

Reply:
It does, but as the current TIAF pipeline is dependent on the existing profile pipeline I'll have to hold off on setting these vars and merging this PR for the time being 👍